content/_data/events/2020-01-30-fosdem.adoc

```diff
@@ -0,0 +1,13 @@
+---
+
+title: "Jenkins at FOSDEM"
+location: "Brussels, Belgium"
+date: "2020-01-30T8:30:00"
```

Comment:
Either we change the date to the "2020-01-31T8:30:00" or we also mention the training here